benchmarks/swe_bench/test_swe_bench.py

```diff
@@ -0,0 +1,131 @@
+import json
```

Comment:
I'm thinking we put this is a `tests/` directory and the way these tests are run is:

```bash
cd benchmarks/swe_bench
pytest
```

As if we end up with dozens of benchmarks we don't really want to pay to test all of them by default.

Reply:
I think the tests/  idea doesn't work unless I make it its own python project.  Otherwise I can't import the files im trying to test from?